apps/admin-ui/src/clients/advanced/AdvancedSettings.tsx

```diff
@@ -36,6 +39,16 @@ export const AdvancedSettings = ({
   const { t } = useTranslation("clients");
   const [open, setOpen] = useState(false);
 
+  const [realm, setRealm] = useState<RealmRepresentation>();
+  const { realm: realmName } = useRealm();
+  const { adminClient } = useAdminClient();
+
+  useFetch(
+    async () => adminClient.realms.findOne({ realm: realmName }),
```

Comment:
No need for this function to be `async`, since it returns a `Promise`.
```suggestion
    () => adminClient.realms.findOne({ realm: realmName }),
```